pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2734,6 +2734,16 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.4.0"
       }
+    },
+    "runtime.win.System.Threading.Thread": {
+      "AssemblyVersionInPackageVersion": {
+        "4.1.0.0": "4.4.0"
+      }
+    },
+    "runtime.unix.System.Threading.Thread": {
+      "AssemblyVersionInPackageVersion": {
+        "4.1.0.0": "4.4.0"
+      }
```

Comment:
I don't think you will need this since you are not doing split packages anymore.